conf.toml

```diff
@@ -1,4 +1,5 @@
 WebStaticFilesPath = "web"
+RunningMode = "prod"
```

Comment:
we call this concept `environment`. however, using a general `environment` config parameter will result in code like next:

```
if (environment == "qa) {
 // new configs will be set here, from now on
}  
else {
 // or here.
}
``` 

which I think is not good pattern as it makes irrelevant all the existing and new config parameters as you can set them in the `if ... else ...` clause mentioned above. We should rather use an specific config parameters and set the relevant value in each deployment configuration, for example,  in this case we could have something like "EnableCorsException", which could be set true in the config for QA deployment only.

Reply:
Changed to use specific config CORSAllowedOrigins instead of depending on environment name.